### PR TITLE
Add site map to footer and card component to local implementation projects

### PR DESCRIPTION
**Pull request by ztocode**

### CODE REVIEW

File: src/js/components/Footer.jsx [hunk 1]
```diff
@@ -1,6 +1,36 @@
 const Footer = () => {
   return (
     <footer className="component Footer">
+      <style>
```
Comment (navneetjoshi223) [inline src/js/components/Footer.jsx:4]: Would be nice if we can maintain all the styling in the Footer.scss

**Review by navneetjoshi223 — COMMENTED** [review]
mostly looks good to me, would just prefer putting all the styling together

File: src/js/components/Footer.jsx (hunk 1 above)
Comment (anaximander) [reply src/js/components/Footer.jsx:4]: I agree we should move those styles to the scss (or we can use inline styles sparingly

**Review by anaximander — APPROVED** [review]
It would be good to move those styles as suggested, but otherwise LGTM 👍

**Pull request merged**